### PR TITLE
Fix std::chrono::duration_cast

### PR DESCRIPTION
**Pull request by emveepee**
Need to use count() to get the value.

### CODE REVIEW

**Review by phunkyfish — APPROVED** [review]

**Comment by phunkyfish** [comment]
Want to increase the version 8.1.0 and add the PVR API changelog note from here: https://github.com/kodi-pvr/pvr.nextpvr/pull/158

Then I can just merge this once the PVR API change is merged.

**Comment by emveepee** [comment]
> 
> 
> Want to increase the version 8.1.0 and add the PVR API changelog note from here: #158
 
Sure there is the issue of changing int64_t to int when I use the count() in this change.  I wanted to see some warnings and I will be adding the  static_cast<int> too

**Pull request merged**
